os_stub/armbuild_lib/divtest.c

```diff
@@ -0,0 +1,15 @@
+/**

```

Comment:
Please just name dev.c, no need to use test.

Reply:
Thanks. I have changed it.